index/generator/schema/schema.go

```diff
@@ -87,6 +88,8 @@ type Schema struct {
 	Resources         []string               `yaml:"resources,omitempty" json:"resources,omitempty"`
 	StarterProjects   []string               `yaml:"starterProjects,omitempty" json:"starterProjects,omitempty"`
 	Git               *Git                   `yaml:"git,omitempty" json:"git,omitempty"`
+	Provider		  string				 `yaml:"provider,omitempty" json:"provider,omitempty"`
```

Comment:
Can you fix the formatting so that the lines above this align?

Reply:
updated